proto/greptime/v1/region/server.proto

```diff
@@ -55,13 +55,19 @@ message RegionRequest {
     AlterRequests alters = 15;
     BulkInsertRequest bulk_insert = 16;
     SyncRequest sync = 17;
+    ListMetadataRequest list_metadata = 18;
   }
 }
 
 message RegionResponse {
   ResponseHeader header = 1;
   uint64 affected_rows = 2;
   map<string, bytes> extensions = 3;
+  // Response for ListMetadataRequest.
+  // It is a json array of region metadatas.
+  // The length of the array is equal to the number of regions in the list request.
+  // The element is null if the region is not found.
+  bytes metadata = 4;
```

Comment:
For the clearness in the future codes, I think it's better to spill it to a type:
```suggestion
  oneof body {
    ListMetadataResponse list_metadata = 4;
  }
}

message ListMetadataResponse {
  bytes metadata = 1;
}
```

Reply:
We haven't released this API, so we can change it later.